frontend/src/common/components/header/Header.tsx

```diff
@@ -81,6 +81,11 @@ const Navbar = ({
                   <NavLink to={SETTINGS_ROUTES.MANAGE}>Settings</NavLink>
                 </li>
               )}
+              {(
```

Comment:
This will only show up whenever the navbar shows up (ie. for cv clients mostly, and staff will not see it if they don't act as company). Is this the intended behaviour?

Reply:
Also it's not showing the actual PR number (blank), and is selected (not sure if this is intended)